packages/asset-swapper/src/utils/market_operation_utils/index.ts

```diff
@@ -165,11 +171,26 @@ export class MarketOperationUtils {
                 this._liquidityProviderRegistry,
             ),
         );
+        const rfqtPromise =
```

Comment:
At what point do we want to short-circuit considering RFQT MMers will not suppot buys to begin with?

Reply:
This is my attempt at making it look nicer: https://github.com/0xProject/0x-monorepo/pull/2555/commits/5effc6ec90e5a90402e0efb01cefd0b6a6a8ef1b